src/index.html

```diff
@@ -9,6 +9,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header 
+      class="header"
```

Comment:
u have to leave atributes on one line if there 1 or 2 of them

Reply:
Rewrote now <header class="header"> in one line.